src/Tree/NodeWrapper.js

```diff
@@ -7,7 +7,8 @@ export default class NodeWrapper extends React.Component {
     enableTransitions: this.props.transitionDuration > 0,
   };
 
-  componentWillReceiveProps(nextProps) {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Thanks for dealing with the warnings on newer React versions 💯  

I really need to put aside time to update these lifecycle methods to `componentDidUpdate` at a minimum.